backend/ecs_tasks/delete_files/requirements.in

```diff
@@ -1,4 +1,4 @@
-pyarrow==1.0.1
+pyarrow==2.0.0
```

Comment:
Upgrading to the latest version didn't solve the issue but I decided to keep it as it adds some performance improvements and fixes.